### DISCUSSION TITLE
Idea: create symbol improvements

### DISCUSSION THREAD

**Discussion by Quentie** [keiwando/toonsquid#347] [Ideas]
Thanks for this cool tool. 
When we currently create a symbol from a timeline layer or a drawing it will automatically create an animation clip in the library with the complete resolution of the project.

It would be cool, if it could take the resolution of the timeline layer / drawing (like it is shown in the timeline) instead (or optionally if this is not useful for others). That way one could animate some assets for a game comfortably together in one timeline and convert them into individual symbols later to export them.

**Comment by Kosdaos1** [comment]
Since I started working in Toonsguid, it was strange to me that it was impossible to make a symbol from the selected frames.
